neuro_flow/cli/batch.py

```diff
@@ -248,3 +248,47 @@ async def clear_cache(config_dir: ConfigDir, batch: str) -> None:
             await runner.clear_cache(None)
         else:
             await runner.clear_cache(batch)
+
+
+@click.command()
+@argument("bake_id", type=BAKE)
+@option(
+    "-a",
+    "--attempt",
+    type=int,
+    default=-1,
+    help="Attempt number, the last attempt by default",
+)
+@option("--local-executor", is_flag=True, default=False, help="Run primary job locally")
+@option(
+    "--from-failed",
+    is_flag=True,
+    default=True,
+    help="Restart from the point of failure",
+)
+@wrap_async()
+async def restart(
+    config_dir: ConfigDir,
+    bake: str,
+    attempt: int,
+    from_failed: bool,
+    local_executor: bool,
+) -> None:
+    """Start a batch.
+
+    Run BATCH pipeline remotely on the cluster.
```

Comment:
Minor: this docstring is same as for `neuro-flow bake` command. I think we should change it somehow.